src/streamlink/plugin/plugin.py

```diff
@@ -323,5 +323,10 @@ def get_streams(self, *args, **kwargs):
     def _get_streams(self):
         raise NotImplementedError
 
+    def stream_title(self):
```

Comment:
Yeah, that seems like a good idea. I really have no problem with it as is, but you're right that it does get changed from None to "Untitled" right now if the plugin has no _get_title function